### PR TITLE
feat(design-tokens): tokenise spacing and color - INNO-1116

### PR DESCRIPTION
**Pull request by degliwe**
# PR description

Please drop a few lines about the PR: what it does, how to test it, etc.

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

* [ ] `package.json` is up-to-date and `@ecl/[system]-base` is part of the dependencies
* [ ] I have checked the dependencies
* [ ] I have given the fractal status “ready” to my component
* [ ] I have declared `@define mycomponent` in the SCSS file
* [ ] I have specified `margin: 0;` on the CSS component
* [ ] I have provided tests
* [ ] I follow the naming guidelines
* [ ] the component supports composition
* [ ] there are no hardcoded strings (all content come from the context)
* [ ] I have filled the README.md file (at least a few lines)
* [ ] My component is listed in the root README
* [ ] My PR has the right label(s)

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 3604011ee82c72854514c54105bbfd9a2dde3a7b

https://deploy-preview-727--europa-component-library.netlify.com

**Review by yhuard — DISMISSED** [review]
Good start 👍 

We will need to create some conventions for the naming of the tokens and the default values in SCSS, let's discuss it on slack.

There's something wrong with the spacing system. Why do you assign non-existing aliases to your themes' spacing aliases?

File: src/systems/ec/ec-design-tokens/themes/theme1/aliases/spacings.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+global:
+  type: all
+  category: spacings
+
+imports:
+  - ../../../tokens/spacings.yml
+
+aliases:
```
[the remark this answers is not in the document]
Comment (raduchiriac) [reply src/systems/ec/ec-design-tokens/themes/theme1/aliases/spacings.yml]: If I remove them I get errors like: `Error converting tokens:  Error: Alias "SPACING_S" not found`

**Comment by raduchiriac** [comment]
This PR is now only focused on the `component-button` and it can be re-reviewed.

Here's a list of things that are intentionally left behind:
- some spacing variables are still not defined inside the buttons https://github.com/ec-europa/europa-component-library/blob/next-v2/src/generic/generic-component-button/_mixins.scss#L3
- some color variables are still not defined inside the buttons
- some files are left in WIP `ec-component-*/tokens.yml` and can be confusing
- main design tokens' names can be a subject of changes, we still have not decided for `COLOR_BLUE_100` or simply `BLUE_100` (or `SPACING_XS` vs. `XS` etc.)

**Review by yhuard — APPROVED** [review]

**Pull request merged**